### PR TITLE
Create Rule “dns-what-and-how-it-works/rule”

### PR DESCRIPTION
**Pull request by Dhruv-0987**
Automatically generated by Decap CMS

### CODE REVIEW

File: rules/dns-what-and-how-it-works/rule.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+type: rule
+title: Do you know DNS, what it is and how it works?
```
Comment (tiagov8) [inline rules/dns-what-and-how-it-works/rule.md:3]: Hi @Dhruv-0987 

What about changing to "Do you know what DNS is and how it works?"

File: rules/dns-what-and-how-it-works/rule.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+created: 2024-02-28T07:04:54.685Z
+guid: 21275f4c-aaf4-4964-9d25-804f3cb56e75
+---
+
+            
+The Domain Name System (DNS) is akin to the internet's phonebook. It's easy to remember a website's name, like www.ssw.com.au, but computers and networks need numerical IP addresses to access websites. DNS translates these user-friendly domain names to IP addresses, allowing browsers to load internet resources.
+
+### DNS explained
+
+`youtube: https://www.youtube.com/embed/{VIDEO_ID}`
```
Comment (tiagov8) [inline rules/dns-what-and-how-it-works/rule.md:17]: Hi @Dhruv-0987 

This link is using a placeholder.

- [ ] 1. Please fix with the proper video ID

File: rules/dns-what-and-how-it-works/rule.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+**Video: Everything You Need to Know About DNS**
```
Comment (tiagov8) [inline rules/dns-what-and-how-it-works/rule.md:19]: Hi @Dhruv-0987 

- [ ] 1. Please add the video lenght at the end as per https://www.ssw.com.au/rules/add-useful-and-concise-figure-captions/#tip-5-add-the-titles--length-on-video-captions

**Review by tiagov8 — CHANGES_REQUESTED** [review]

File: rules/dns-what-and-how-it-works/rule.md (hunk 1 above)
Comment (Dhruv-0987) [reply rules/dns-what-and-how-it-works/rule.md]: Hi @tiagov8 thanks for the feedback I have changed it so.

File: rules/dns-what-and-how-it-works/rule.md (hunk 2 above)
Comment (Dhruv-0987) [reply rules/dns-what-and-how-it-works/rule.md]: Fixed it with the proper video id.

File: rules/dns-what-and-how-it-works/rule.md (hunk 3 above)
Comment (Dhruv-0987) [reply rules/dns-what-and-how-it-works/rule.md]: ✅ Done - added video length at the end.

**Comment by Dhruv-0987** [comment]
This PR was set to close automatically when I changed the branch name and I had to create a new PR from the new branch, its being continued in the PR: https://github.com/SSWConsulting/SSW.Rules.Content/pull/8020

**Pull request closed without merging**